xbmc/pvr/channels/PVRChannelGroups.cpp

```diff
@@ -71,6 +71,9 @@ bool CPVRChannelGroups::Update(const CPVRChannelGroup &group, bool bUpdateFromCl
     if (group.GroupID() > 0)
       updateGroup = GetById(group.GroupID());
 
+    if (!updateGroup && group.IsInternalGroup() && (updateGroup = GetGroupAll()) && bUpdateFromClient)
```

Comment:
> returns and continues are evil

that's a matter of taste, but @opdenkamp is the maintainer of the PVR part and we should respect his coding-style ;)